mock-mhs-adaptor/src/main/java/uk/nhs/adaptors/mockmhsservice/service/MockMhsService.java

```diff
@@ -19,71 +15,72 @@
+        String odsCode) {
         headers.setContentType(MediaType.TEXT_HTML);
         objectMapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, true);
         objectMapper.configure(DeserializationFeature.FAIL_ON_NULL_FOR_PRIMITIVES, true);
 
         if (odsCode.isEmpty()) {
             LOGGER.error("Missing ods-code header");
-            return new ResponseEntity<>(internalServerErrorResponse, headers, HttpStatus.BAD_REQUEST);
+            return new ResponseEntity<>(INTERNAL_SERVER_ERROR_RESPONSE, headers, HttpStatus.BAD_REQUEST);
         }
 
         if (!waitForResponse.equals("false")) {
             LOGGER.error("Missing or invalid wait-for-response header");
-            return new ResponseEntity<>(internalServerErrorResponse, headers, HttpStatus.BAD_REQUEST);
+            return new ResponseEntity<>(INTERNAL_SERVER_ERROR_RESPONSE, headers, HttpStatus.BAD_REQUEST);
         }
 
         try {
             verifyOutboundMessagePayload(mockMhsMessage);
         } catch (MockMHSException e) {
             LOGGER.error(e.getMessage(), e);
-            return new ResponseEntity<>(internalServerErrorResponse, headers, INTERNAL_SERVER_ERROR);
+            return new ResponseEntity<>(INTERNAL_SERVER_ERROR_RESPONSE, headers, INTERNAL_SERVER_ERROR);
         } catch (JsonProcessingException e) {
             LOGGER.error("Error content of request body does not match expected JSON", e);
-            return new ResponseEntity<>(internalServerErrorResponse, headers, INTERNAL_SERVER_ERROR);
+            return new ResponseEntity<>(INTERNAL_SERVER_ERROR_RESPONSE, headers, INTERNAL_SERVER_ERROR);
         }
 
-        if (interactionId.equals(mockValidInteractionId)) {
+        if (interactionId.equals(EXTRACT_CORE_INTERACTION_ID)) {
             try {
-                var inboundMessage = stubInboundMessage.replace("%%ConversationId%%", correlationId);
+                var inboundMessage = STUB_CONTINUE_REPLY_INBOUND_MESSAGE.replace("%%ConversationId%%", correlationId);
                 inboundProducer.sendToMhsInboundQueue(inboundMessage);
                 LOGGER.info("Placed message on Inbound Queue, conversationId: " + correlationId);
                 headers.setContentType(MediaType.TEXT_XML);
-                return new ResponseEntity<>(stubEbXmlResponse, headers, ACCEPTED);
+                return new ResponseEntity<>(STUB_ACCEPTED_RESPONSE, headers, ACCEPTED);
             } catch (JmsException e) {
                 LOGGER.error("Error could not produce inbound reply", e);
-                return new ResponseEntity<>(internalServerErrorResponse, headers, INTERNAL_SERVER_ERROR);
+                return new ResponseEntity<>(INTERNAL_SERVER_ERROR_RESPONSE, headers, INTERNAL_SERVER_ERROR);
             }
+        } else if (interactionId.equals(ACKNOWLEDGEMENT_INTERACTION_ID)) {
+            LOGGER.info(String.format("Message acknowledgement accepted, conversationId: %s", correlationId));
```

Comment:
Don't explicitly log conversationId. This is part of our logback.xml format and it always included.